### ISSUE TITLE
ECMAScript Module

### ISSUE THREAD

**Issue by samueldmeyer** [mixpanel/mixpanel-js#340]
When using Webpack (as a part of Angular), I get a warning on each build about 

`Warning: ..../mixpanel.service.ts depends on 'mixpanel-browser'. CommonJS or AMD dependencies can cause optimization bailouts.`

I currently import using

`import * as mixpanel from 'mixpanel-browser';`

It would be nice to have an ECMAScript module so that it doesn't cause optimization bailouts.

**Comment by tdumitrescu** [comment]
Does it give you the same warning if you use the recommended import style?
```js
import mixpanel from 'mixpanel-browser';
```

It is also possible to import directly from the source module like in https://github.com/mixpanel/mixpanel-js/tree/master/examples/es2015-babelify:
```js
import mixpanel from 'mixpanel-browser/src/loader-module';
```

**Comment by samueldmeyer** [comment]
Yes, the recommended import style gives the same warning. Importing directly from the source module causes a Typescript error, as no types are defined for that file.

I haven't converted from CommonJS to ESM myself, but it will require at least adding

```
"type": "module"
```

to package.json. A quick reference I found: https://medium.com/digio-australia/migrating-an-npm-package-to-use-es-modules-d59877963d61
